src/odemis/gui/main_xrc.py

```diff
@@ -5802,6 +5805,64 @@ def __init_resources():
                   <orient>wxVERTICAL</orient>
                   <object class="sizeritem">
                     <object class="FoldPanelBar" name="fpb_settings">
+                    <object class="FoldPanelItem">
```

Comment:
All these changes should be done in main.xrc, and then generated into main_xrc.py, so you should always push changes to both files at the same time.

Reply:
I will fix it as soon as the other pr is merged :)